### PR TITLE
Change copyright to GraphQL Contributors, License to MIT

### PR DESCRIPTION
**Pull request by acao**

### CODE REVIEW

**Comment by acao** [comment]
@caniszczyk @benjie here's the new and improved one, with just new copyright messages

File: packages/graphiql/example/schema.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /**
- *  Copyright (c) Facebook, Inc. and its affiliates.
+ *  Copyright (c) 2019 GraphQL Contributors and its affiliates.
```
Comment (benjie) [inline packages/graphiql/example/schema.js]: `and its affiliates` has started creeping through here, @acao

**Review by benjie — APPROVED** [review]
Other than a few extraneous `and its affiliates`; this all looks good 👍

File: packages/graphiql/example/schema.js (hunk 1 above)
Comment (acao) [reply packages/graphiql/example/schema.js]: ope!

**Comment by acao** [comment]
@benjie thanks! made that tweak, forgot to mention

**Comment by acao** [comment]
TBH, it feels a bit sketchy making this PR against a fork of mine rather than an `ide-monorepo` branch of the official repository, but oh well!

**Comment by benjie** [comment]
Oh you're right; this won't be stored in the main issue tracker - the main monorepo PR will (because it's a PR on the GraphiQL repo), but this is a PR to a PR and if you delete this repo its history will be lost.

Let's move the monorepo to a graphiql branch as you suggested and re-open the PR there.

**Comment by acao** [comment]
ok, good idea!

**Comment by acao** [comment]
Moved here: https://github.com/graphql/graphiql/pull/849

**Pull request closed without merging**